code/datums/emotes.dm

```diff
@@ -31,9 +31,9 @@
 	if(!message_mommi)
 		message_mommi = message_robot
 
-/datum/emote/proc/run_emote(mob/user, params, type_override)
+/datum/emote/proc/run_emote(mob/user, params, type_override, ignore_status = FALSE)
 	. = TRUE
-	if(!can_run_emote(user))
+	if(!can_run_emote(user) || (ignore_status && !can_run_emote(user, FALSE)))
```

Comment:
Can't you just do `if(!can_run_emote(user, ignore_status))`?

Reply:
Since ignore_status defaults to FALSE (as in, don't ignore status; kind of confusing), it would be
```if(!can_run_emote(user, !ignore_status))``` but yes, good catch.